src/test/validation_flush_tests.cpp

```diff
@@ -20,7 +20,7 @@ BOOST_AUTO_TEST_CASE(getcoinscachesizestate)
 {
     CTxMemPool mempool;
     BlockManager blockman{};
-    CChainState chainstate{mempool, blockman};
+    CChainState chainstate{&mempool, blockman};
```

Comment:
Any idea why there's a different `tx_pool` from `mempool` in this test? It's not used anywhere so presumably can be removed?

Reply:
Yep, looks like it. I think when I wrote these I wanted to cover the fact that you could pass in a mempool reference that wasn't owned by the chainstate, but I don't think there's any point in doing that now.